mkdocs.yml

```diff
@@ -0,0 +1,17 @@
+site_name: ModelGauge
+repo_url: https://github.com/mlcommons/modelgauge/
+edit_uri: blob/main/docs/
+theme:
+  name: readthedocsgit st
+plugins:
+  - search
+nav:
+  - index.md
+  - dev_quick_start.md
```

Comment:
Do these need to say they are in the "docs" folder?

Reply:
Nope, mkdocs just uses the docs folder by default.